.github/workflows/ci-libshortfin.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         repository: iree-org/iree
         path: ${{ env.IREE_REPO_DIR }}
         submodules: false
-        ref: iree-3.1.0rc20241220
+        ref: iree-3.1.0rc20250103
```

Comment:
shortfin unit tests seem to be hanging, even on standard github-hosted runners running CPU tests: https://github.com/nod-ai/shark-ai/actions/runs/12604999242/job/35132947139?pr=747